indy_node/server/domain_req_handler.py

```diff
@@ -422,6 +426,122 @@ def handleGetRevocRegDefReq(self, request: Request):
                                   proof=proof)
         return result
 
+    def handleGetRevocRegReq(self, request: Request):
+        req_ts = request.operation.get(TIMESTAMP)
+        revoc_reg_def_id = request.operation.get(REVOC_REG_DEF_ID)
+        author_did = request.identifier
+        # Get root hash corresponding with givtimestamp
+        past_root = self.tsRevoc_store.get_equal_or_prev(req_ts)
+        # Path to corresponding REVOC_REG_ENTRY
+        path = domain.make_state_path_for_revoc_reg_entry(authors_did=author_did,
+                                                          revoc_reg_def_id=revoc_reg_def_id)
+        if past_root is None:
+            reply_data = None
+            seq_no = None
+            last_update_time = None
+            proof = None
+        else:
+            encoded_entry = self.state.get_for_root_hash(past_root, path)
+            revoc_reg_entry, seq_no, last_update_time = domain.decode_state_value(encoded_entry)
+
+            reply_data = None if revoc_reg_entry is None else \
+                {
+                    REVOC_REG_DEF_ID: revoc_reg_def_id,
+                    REVOC_TYPE: revoc_reg_entry.get(REVOC_TYPE),
+                    VALUE: {
+                        ACCUM: revoc_reg_entry[VALUE].get(ACCUM)
+                    }
+                }
+            proof = self.make_proof(path, head_hash=past_root)
```

Comment:
A separate path and entry needs to be used for GET_REVOC_REG in the state trie, otherwise the state proof can not be verified. The VALUE must be the same as the one stored in the state trie (here we store ACCUM+ISSUED/REVOKED, while return ACCUM only).
Also please add tests that state proofs in replies to GET_REVOC_REG and GET_REVOC_REG_DELTA can be verified.